Realm/RLMSyncUser.h

```diff
@@ -173,6 +179,22 @@ NS_SWIFT_UNAVAILABLE("Use the full version of this API.");
  */
 - (void)changePassword:(NSString *)newPassword forUserID:(NSString *)userID completion:(RLMPasswordChangeStatusBlock)completion;
 
+#pragma mark - Administrator API
+
+/**
+ Given a Realm Object Server authentication provider and a provider identifier for a user
+ (for example, a username), look up and return user information for that user.
+
+ @param provider    The authentication provider that manages the user whose information is desired.
+ @param providerID  The identity of the user as issued by the authentication provider.
+ @param completion  Completion block invoked when request has completed or failed.
+                    The callback will be invoked on a background queue provided
+                    by `NSURLSession`.
+ */
+- (void)retrieveUserInfoForProvider:(RLMIdentityProvider)provider
+                   providerIdentity:(NSString *)providerID
+                         completion:(RLMRetrieveUserBlock)completion;
```

Comment:
I'm not sure I like this naming. `providerIdentity` suggests to me that parameter is the identity of the provider, when in fact it's the provider's concept of the identity of the user. Perhaps `-retrieveInfoForUser:identityProvider:completion:`?